scripts/jenkins-mender-convert-build.sh

```diff
@@ -0,0 +1,307 @@
+
+         pr_status_endpoint=https://api.github.com/repos/mendersoftware/$repo/statuses/$git_commit
+         set -x
+         curl --user "$GITHUB_BOT_USER:$GITHUB_BOT_PASSWORD" \
+              -d "$request_body" \
+              "$pr_status_endpoint"
+         set +x
+done
+
+cd -
+
+}
+
+github_PR_status "pending" "Mender convert acceptance tests has started"
+
+# Wait unti build host is ready.
+attempts=180
+while [ $attempts -gt 0 ] && ! systemctl is-system-running; do
+    # Wait for init-script to finish.
+    sleep 10
+    attempts=$(expr $attempts - 1 || true)
+done
+sudo journalctl -u rc-local | cat || true
+ if [ $attempts -le 0 ]; then
+    exit 1
+fi
+
+
+echo "deb http://us.archive.ubuntu.com/ubuntu/ bionic main restricted universe" | sudo tee -a /etc/apt/sources.list
+echo "deb http://security.ubuntu.com/ubuntu bionic-security main restricted universe" | sudo tee -a /etc/apt/sources.list
+
+sudo apt-get update
+sudo apt-get -qy --force-yes install e2fsprogs=1.44.1-1
+sudo apt-get -qy --force-yes install kpartx bison unzip mtools parted mtd-utils u-boot-tools pigz flex
+sudo apt-get -qy --force-yes install python-pip
+sudo pip2 install pytest --upgrade
+sudo pip2 install pytest-xdist --upgrade
+sudo pip2 install pytest-html --upgrade
+sudo pip2 install -I fabric==1.14.0
+
+cat >> ~/.mtoolsrc << EOF
+mtools_skip_check=1
+EOF
+
+get_arm_compiler
+if [ ! -e ${LINARO_BINARY_PATH} ]; then
+    echo "${LINARO_BINARY_PATH}: can not be found"
+    github_PR_status "failure" "Cannot setup arm compiler"
+    exit 1
+fi
+
+get_debian
+if [ ! -e ${DEBIAN_IMAGE} ]; then
+    echo "${DEBIAN_IMAGE}: can not be found"
+    github_PR_status "failure" "Cannot find debian image"
+    exit 1
+fi
+
+get_raspbian
+if [ ! -e ${RASPBIAN_IMAGE} ]; then
+    echo "${RASPBIAN_IMAGE}: can not be found"
+    github_PR_status "failure" "Cannot find raspbian image"
+    exit 1
+fi
+
+export PATH=$PATH:$(pwd)/gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf/bin
+
+build_mender_client
+if [ ! -e $GOPATH/bin/mender-$host ]; then
+    echo "$GOPATH/bin/mender-$host: can not be found"
+    exit 1
+fi
+if [ ! -e $GOPATH/bin/mender-arm ]; then
+    echo "$GOPATH/bin/mender-arm: can not be found"
+    exit 1
+fi
+
+build_mender_artifact
+if [ ! -e $GOPATH/bin/mender-artifact ]; then
+    echo "mender-artifact: can not be found"
+    github_PR_status "failure" "Cannot find mender artifact"
+    exit 1
+fi
+
+export PATH=$PATH:$GOPATH/bin
+
+if ! [ -x "$(command -v mender-artifact)" ]; then
+    echo "mender-artifact: not found in PATH."
+    github_PR_status "failure" "mender-artifact: not found in PATH."
+    exit 1
+fi
+
+build_qemux86_64_vanilla_image || rc=$?
+
+[[ $rc -eq 0 ]] || { exit 1; }
+
+cd $WORKSPACE/mender-convert
+
+./mender-convert from-raw-disk-image --raw-disk-image $QEMUX86_64_RAW_DISK_IMAGE --mender-disk-image ${CONVERTED_QEMUX86_64_IMAGE} \
+    --device-type qemux86_64 --mender-client $GOPATH/bin/mender-$host --artifact-name release-1_1.5.0 \
```

Comment:
Maybe it would be worth making `--artifact-name` value as a parameter, so that it reflects mender client version?

Reply:
I see it added in the following commit, so mark as resolved.